### PR TITLE
docs: load colocated docs from app-management and cache-cleaner

### PR DESCRIPTION
**Pull request by varl**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -508,8 +509,8 @@ nav:
             - Email: '@github(dhis2/dhis2-docs, src/developer/web-api/email.md, master)'
             - Data Store: '@github(dhis2/dhis2-docs, src/developer/web-api/data-store.md, master)'
             - Org unit profile: '@github(dhis2/dhis2-docs, src/developer/web-api/organisation-unit-profile.md, master)'
-            - Apps: '@github(dhis2/dhis2-docs, src/developer/web-api/apps.md, master)'
-        Loading Apps: '@github(dhis2/dhis2-docs, src/developer/apps.md, master)'
+            - '@github(dhis2/app-management-app, docs/developer/index.yml,master)'
+        '@github(dhis2/app-management-app, docs/developer/index.yml,master)'
```
Comment (Philip-Larsen-Donnelly) [inline mkdocs.yml:513]: what is the difference between this and the entry above?

Comment (varl) [reply mkdocs.yml:513]: Ah, thanks. I missed that one of the original lines is for the `web-api/apps.md`.

Comment (varl) [inline mkdocs.yml:513]:
```suggestion
            - Apps: '@github(dhis2/dhis2-docs, src/developer/web-api/apps.md, master)'
        '@github(dhis2/app-management-app, docs/developer/index.yml,master)'
```

Comment (varl) [reply mkdocs.yml:513]: Fixed it in 17f7cc4

**Review by Philip-Larsen-Donnelly — APPROVED** [review]

**Pull request merged**